docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,255 @@
+
+### Add Mozilla Maven Repository
+
+In order to be able to download Glean, we need to add Mozilla’s maven repository URL to the Project `build.gradle` file.  In
+Android Studio, for our project named “GleanCrashExample”, there will be a `build.gradle` that has `(Project: GleanCrashExample)`
+next to it.  In this Gradle file, you will find a section called `allProjects` which has in it a section called `repositories`.
+You may already find `google()` and `jcenter()` there, but we need to add the following in order to download Glean:
+
+```Groovy
+maven {
+    url "https://maven.mozilla.org/maven2"
+}
+```
+
+### Add Python Environment
+
+We add a Python environment for the [glean_parser](https://github.com/mozilla/glean_parser/) to run in with the following lines,
+added to the very top of the `build.gradle` file:
+
+```Groovy
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
+```
+
+### Import SDK Generator Script
+
+The last part of the changes to the app’s `build.gradle` file is to include the `sdk_generator.gradle` script.  Right before the
+end of the `build.gradle` file, add the following (making sure the version here matches the version of the `implementation`
+line above):
+
+```Groovy
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/v16.0.0/components/service/glean/scripts/sdk_generator.gradle'
+```
+
+That should be it for additions to the build configuration files.  You should try to do a Gradle sync now to make sure
+everything works.  If you have any problems, try going back through the configuration steps to make sure you got everything.
+
+## Instrument The Project With Glean
+
+### Initial Integration
+
+Now that we have added Glean to our app, we need to do a little initialization in order to get it ready to record and send
+data. At the top of your `MainActivity.kt` file, add the following import statement:
+
+```Kotlin
+import mozilla.components.service.glean.Glean
+```
+
+Then inside of the `onCreate()` function of the `MainActivity`, add the following 2 lines to enable and initialize Glean:
+
+```Kotlin
+Glean.setUploadEnabled(true)
+Glean.initialize(this)
+```
+
+That’s it!  The app now has Glean installed and enabled and will be able to send the baseline ping without any additional work!
+
+### Add Custom Metric
+
+Since we are instrumenting crashes with some custom metrics, we still have some work to do.  We will need to add a
+`metrics.yaml` file to define the metrics we will use to record our crash information and a `pings.yaml` file to define a
+custom ping which will give us some control over the scheduling of the uploading of the crash telemetry.
+
+In order to do this we need to make a decision.  Mainly, what metric type will we use to represent our crash data.  This could
+probably be argued several ways, but I have chosen to instrument it as an [event](https://mozilla.github.io/glean/book/user/metrics/event.html),
```

Comment:
Make it a relative link to the .md file and our linkchecker will be happy to ensure it keeps working